pkg/protocol/sybilprotection/sybilprotectionv1/sybilprotection.go

```diff
@@ -116,7 +116,7 @@ func (o *SybilProtection) CommitSlot(slot iotago.SlotIndex) (committeeRoot, rewa
 
 	// TODO: check if the following value is correctly set to twice eviction age
 	// maxCommittableSlot = 2 * evictionAge
```

Comment:
Maybe fix the comment as well.

Reply:
It is set correctly now so TODO is done.